src/components/chip-set/chip-set.tsx

```diff
@@ -337,35 +340,60 @@ export class ChipSet {
     }
 
     public render() {
-        if (this.type === 'input') {
-            return this.renderInputChips();
-        }
-
         const classes = {
             'mdc-chip-set': true,
-            disabled: this.disabled || this.readonly,
             'mdc-text-field--with-trailing-icon': true,
+            disabled: this.disabled || this.readonly,
         };
+
         if (this.type) {
             classes[`mdc-chip-set--${this.type}`] = true;
         }
 
-        const chipSetLabel = this.renderChipSetLabel();
-        if (chipSetLabel) {
-            classes['chip-set--with-label'] = true;
+        if (this.type === 'input') {
+            Object.assign(classes, {
+                'mdc-text-field': true,
+                'mdc-text-field--outlined': true,
+                'mdc-chip-set--input': true,
+                'lime-text-field--readonly': this.readonly,
+                'has-chips': this.value.length !== 0,
+                'has-leading-icon': this.leadingIcon !== null,
+                'has-clear-all-button': this.clearAllButton,
+            });
         }
 
         const value = this.getValue();
 
-        return (
-            <div class={classes} role="grid">
-                {chipSetLabel}
-                {value.map(this.renderChip)}
-                {this.renderHelperLine()}
-            </div>
-        );
+        return [
+            <limel-notched-outline
+                role="grid"
+                labelId={this.labelId}
+                label={this.label}
+                required={this.required}
+                invalid={this.invalid || this.isInvalid()}
+                disabled={this.disabled}
+                readonly={this.readonly}
+                hasValue={!!this.value}
+                hasLeadingIcon={!!this.leadingIcon}
+                hasFloatingLabel={this.floatLabelAbove()}
+                onClick={this.handleTextFieldFocus}
+            >
+                <div slot="content" class={classes}>
+                    {this.renderContent(value)}
+                </div>
+            </limel-notched-outline>,
+            this.renderHelperLine(),
```

Comment:
I've intentionally moved the helper-line here. Previously, only `type` `input` would show the helper text. But this way other types of `chip-set` can also show the helper text. I believe this is the correct and consistent way of doing it. There is no point for the component to have a prop, that only works for a certain type.